README.md

```diff
@@ -649,6 +649,15 @@ and couchdb-lucene will authenticate to couchdb.
 
 A couple of 'expert' options can be set in the couchdb-lucene.ini file;
 
+CouchDB indexing documents can be blacklisted. This is useful when using 2 C-L instances,
+one indexing a new version of an index, allowing the old one to get the latest updates while the new one.
+is being indexed.
+<pre>
+[couchdb]
+blacklist=gin_call_centre/_design/search-version:0.0.5
+blacklist=gin_call_centre/_design/search-version:0.0.6
+</pre>
```

Comment:
Since this is loaded into a single CouchDB instance, how does either c-l instances which blacklist to use?